mobile/src/components/setting/setting.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+import { View, Text } from 'react-native';
```

Comment:
We have our own components for this
```suggestion
import { View, Text } from '~/components/common/common';
```